test/integration/default/verify/controls/aws_iam_root_user.rb

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Extra blank line :smile: 

Sorry...I'm feeling like this review is super nit picky.

The actual code is looking very good though!